rebar.config.erl

```diff
@@ -330,6 +308,7 @@ relx_apps(ReleaseType, Edition) ->
         compiler,
         runtime_tools,
         redbug,
+        xmerl,
```

Comment:
Q: is the intention to change the order of starting/loading this indeed?  It was one of the last ones before.

Reply:
this application can be started in any order.